tests/phpunit/unit/MediaWiki/Request/EditRequestParserTest.php

```diff
@@ -164,14 +164,14 @@ public function provideUnsupportedEntityVersion(): iterable {
 
 	/** @dataProvider provideInvalidReconcileJson */
 	public function testParseRequestInterface_invalidReconcileJson( string $reconcile ): void {
```

Comment:
This test doesn’t make sense anymore – the request parser no longer decodes JSON.

Reply:
done